### ISSUE TITLE
How to report custom loss function and metrics during an experiment

### ISSUE THREAD

**Issue by jac99** [comet-ml/comet-quickstart-guide#6]
I'm not sure if this is the right place to post this question.
I've just started to use comet-ml and it's a great product. I was always wanted to have such functionality.

In Keras I'm training a network using a custom loss function and custom metrics.
My model.compile()  looks like this:
`model.compile(loss=contrastive_loss, metrics=[my_custom_metric1, my_custom_metric2])`

When the network is training, custom loss and metrics are not reported on the comet-ml experiment site. I can see these values in **stdout** section (as printed by Keras on the terminal). But **metrics** section and **graph** is empty.

How can I report and graph these values during the training. Documentation mentions using experiment.log_metric() function. But where/how do I call this function. Should I call this function from within my custom loss and metric functions? So the custom loss and metrics is reported for each minibatch?

Or is there a way to automatically report and graph custom loss and metrics passed as arguments to model.compile?

**Comment by gidim** [comment]
Where do you print your metrics to stdout? is it done automatically by keras?
If so, it should also work with Comet.ml. I'd be happy to help you fix this. Can you share your code so I could reproduce? You're also welcome to email it to us support@comet.ml.
